app/views/catalog/_search_form.html.erb

```diff
@@ -10,7 +10,7 @@
     <div class="input-group col-sm-9">
       <%= text_field_tag :q, current_search_parameters , class: "q form-control", id: "search-field-header", placeholder: t("hyrax.search.form.q.placeholder") %>
 
-      <div class="input-group-btn">
+      <div class="input-group-append">
```

Comment:
@Dananji Looks like this is a nested `input-group`, which could be messing with things?  I wonder if we should approach this by looking at the markup surrounding the search forms themselves to see if they're aligned with BS4?

Reply:
Maybe this route?  https://getbootstrap.com/docs/4.0/components/forms/#inline-forms